rarible/src/exchange-v-1.ts

```diff
@@ -0,0 +1,117 @@
+import { Address, BigInt, Bytes, TypedMap } from "@graphprotocol/graph-ts";
+import { ExchangeCall } from "../generated/ExchangeV1/ExchangeV1";
+import * as airstack from "./modules/airstack";
+import { Transaction } from "../generated/schema";
+// import { isNFT } from "./rarible-helper";
+
+export function getOrCreateTransaction(
+  hash: Bytes,
+  nftAddress: Address,
+  nftId: BigInt
+): Transaction {
+  let entity = Transaction.load(
+    hash.toHexString() +
```

Comment:
Please extract logic of creating id in a different function/variable & reuse same on line 20/